extensions/azurecore/src/azurecore.d.ts

```diff
@@ -273,17 +274,6 @@ declare module 'azurecore' {
 		PATCH
 	}
 
-	/**
-	 * Custom version of NetworkResponse from @azure\msal-common\dist\network\NetworkManager.d.ts
-	 * with body renamed to data to avoid breaking changes with extensions. See
-	 * https://github.com/microsoft/azuredatastudio/pull/22761 for details.
```

Comment:
You've verified that everything is still good with the new response type?

Reply:
Yep, I went through the migration flow and everything worked successfully for me.